releng - drone docs build affected by setuptools_scm

```
$ make ghpages
git checkout gh-pages && \
mv docs/build/html new-docs && \
rm -rf docs && \
mv new-docs docs && \
git add -u && \
git add -A && \
git commit -m "Updated generated Sphinx documentation"
error: The following untracked working tree files would be overwritten by checkout:
	.eggs/README.txt
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/LICENSE
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/PKG-INFO
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/RECORD
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/WHEEL
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/entry_points.txt
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/requires.txt
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/top_level.txt
	.eggs/setuptools_scm-3.4.3-py3.7.egg/EGG-INFO/zip-safe
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/__init__.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/__main__.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/config.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/discover.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/file_finder.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/file_finder_git.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/file_finder_hg.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/git.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/hacks.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/hg.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/integration.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/utils.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/version.py
	.eggs/setuptools_scm-3.4.3-py3.7.egg/setuptools_scm/win_py31_compat.py
Please move or remove them before you switch branches.
Aborting
make: [ghpages] Error 1 (ignored)
Makefile:25: recipe for target 'ghpages' failed
```

--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,7 @@
 *.pyc
 *.egg-info/
 *.swp
+.eggs/
 c7n/version.py
 src/
 deps/

